python/ceed_vector.py

```diff
@@ -58,6 +58,17 @@ def __str__(self):
 
         return out_string
 
+    # Copy self to a different Vector
+    def copy(self, vec_copy):
+        """Copies the array of a self into the array of vec_copy.
+
+           Args:
+             *vector: the Vector to copy to"""
+
+        # libCEED call
+        err_code = lib.CeedVectorCopy(self._pointer[0], vec_copy._pointer)
+        self._ceed._check_error(err_code)
+
```

Comment:
Then you can do this
```suggestion
             *vector: the Vector to copy"""

        # libCEED call
        err_code = lib.CeedVectorCopy(self._pointer[0], vec_copy._pointer[0])
        self._ceed._check_error(err_code)

```